src/jace/api.py

```diff
@@ -72,18 +75,19 @@ def jit(
         fun: Function to wrap.
         primitive_translators: Use these primitive translators for the lowering to SDFG.
             If not specified the translators in the global registry are used.
-        kwargs: Jit arguments.
+        kwargs: Jit arguments, see `JITOptions` for more.
```

Comment:
```suggestion
        kwargs: jit arguments, see `JITOptions` for more.
```

Reply:
We should be consistent and write JIT capital or `Jit` as it is the abbreviation of "Just in time" ('Just' is capital because it is the beginning of a sentence).

Reply:
I'd rather use `JIT` then but it's up to you...

Reply:
I'd rather use `JIT` then but it's up to you...